models/prod/final/final_gdgs_2024_test.yml

```diff
@@ -0,0 +1,108 @@
+models:
+  - name: farmer_endline
+    description: "Farmer data compiled with Endline Status from encounters"
+    columns:
+      - name: farmer_id
+        tests:
+          - not_null
+      - name: farmer_name
+        tests:
+          - not_null
+      - name: dam
+        tests:
+          - not_null
+      - name: district
+        tests:
+          - not_null
+      - name: subject_type
+        tests:
+          - not_null
+      - name: state
+        tests:
+          - not_null
+      - name: taluka
+        tests:
+          - not_null
+      - name: village
+        tests:
+          - not_null
+      - name: category_of_farmer
+        tests:
+          - accepted_values:
+              values:
+                - 'Marginal (0-2.49 acres)'
+                - 'Small (2.5-4.99 acres)'
+                - 'Semi-medium (5 to 9.99 acre)'
+                - 'Medium (10-24.99 acres)'
+                - 'Large (above 25 acres)'
+                - 'Widow'
+                - 'Disabled'
+                - 'Family of farmer who committed suicide'
+                - 'NULL'
```

Comment:
is there a null?

Reply:
Not yet, but I thought to keep it incase there's a null in the future - do I remove it and let it fail the test if it does happen in the future?